src/accelerate/accelerator.py

```diff
@@ -306,11 +307,11 @@ def __init__(
             deepspeed_plugin.set_mixed_precision(mixed_precision)
             deepspeed_plugin.set_deepspeed_weakref()
 
-        if os.environ.get("ACCELERATE_USE_FSDP", "false") == "true" or isinstance(
-            fsdp_plugin, FullyShardedDataParallelPlugin
-        ):
-            if is_torch_version("<", FSDP_PYTORCH_VERSION):
-                raise ValueError(f"FSDP requires PyTorch >= {FSDP_PYTORCH_VERSION}")
+        # if os.environ.get("ACCELERATE_USE_FSDP", "false") == "true" or isinstance(
+        #     fsdp_plugin, FullyShardedDataParallelPlugin
+        # ):
+        # if is_torch_version("<", FSDP_PYTORCH_VERSION):
+        #     raise ValueError(f"FSDP requires PyTorch >= {FSDP_PYTORCH_VERSION}")
```

Comment:
Here I need to adjust for the FSDP version, as it doesn't quite work on the `.dev` related additionals when using the docker image

Reply:
Relying on https://github.com/huggingface/accelerate/pull/3024 to fix